api/go.mod

```diff
@@ -5,16 +5,16 @@ go 1.16
 require (
 	github.com/google/gofuzz v1.2.0
 	github.com/metal3-io/ip-address-manager/api v0.0.0-20211018090204-6be1b3878f19
-	github.com/onsi/gomega v1.15.0
+	github.com/onsi/gomega v1.16.0
 	github.com/pkg/errors v0.9.1
 	golang.org/x/net v0.0.0-20210520170846-37e1c6afe023
-	k8s.io/api v0.21.4
-	k8s.io/apimachinery v0.21.4
-	k8s.io/client-go v0.21.4
-	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
-	sigs.k8s.io/cluster-api v0.4.4
-	sigs.k8s.io/controller-runtime v0.9.7
+	k8s.io/api v0.22.2
+	k8s.io/apimachinery v0.22.2
+	k8s.io/client-go v0.22.2
+	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
+	sigs.k8s.io/cluster-api v1.0.0
+	sigs.k8s.io/controller-runtime v0.10.2
```

Comment:
Strange, I also had a comment to update this too `v0.10.3-0.20211011182302-43ea648ec318` following CAPI, I don't know how it got deleted.

Reply:
Done